automatic/gnupg/gnupg.nuspec

```diff
@@ -0,0 +1,65 @@
+    <owners>chocolatey, ludicrousByte, wget</owners>
+    <title>GnuPG</title>
+    <authors>Werner Koch, The GnuPG Project</authors>
+    <projectUrl>https://www.gnupg.org/</projectUrl>
+    <iconUrl>https://cdn.jsdelivr.net/gh/chocolatey/chocolatey-coreteampackages@901944b6fe60360ef2764c9fc53fe69dee99abd5/icons/gnupg.png</iconUrl>
+    <copyright>Copyright 1998-2019 The GnuPG Project</copyright>
+    <licenseUrl>https://www.gnu.org/licenses/gpl-3.0.html</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <docsUrl>https://www.gnupg.org/documentation/manuals/gnupg/</docsUrl>
+    <mailingListUrl>https://lists.gnupg.org/pipermail/gnupg-users/</mailingListUrl>
+    <bugTrackerUrl>https://bugs.gnupg.org/gnupg/index</bugTrackerUrl>
+    <tags>gnupg-modern gnupg admin pgp openpgp encrypt rfc4880 security aes rsa signature</tags>
+    <summary>GnuPG is a complete and free implementation of the OpenPGP standard as defined by RFC4880 (also known as PGP)</summary>
+    <description><![CDATA[GnuPG itself is a commandline tool without any graphical stuff. It is the real
+crypto engine which can be used directly from a command prompt, from shell
+scripts or by other programs. Therefore it can be considered as a backend
+for other applications.
+
+However, even when used on the command line it provides all functionality
+needed - this includes an interactive menu system. The set of commands
+of this tool will always be a superset of those provided by any frontends.
+
+## Features
+
+- Full replacement of PGP.
+- Does not use any patented algorithms.
+- GPLed, written from scratch.
+- Can be used as a filter program.
+- Full OpenPGP implementation (see RFC4880 at RFC Editor).
+- Better functionality than PGP and some security enhancements over PGP 2.
+- Decrypts and verifies PGP 5, 6 and 7 messages.
+- Supports ElGamal, DSA, RSA, AES, 3DES, Blowfish, Twofish, CAST5, MD5, SHA-1,
+  RIPE-MD-160 and TIGER.
+- Easy implementation of new algorithms using extension modules.
+- The User ID is forced to be in a standard format.
+- Supports key and signature expiration dates.
+- English, Danish, Dutch, Esperanto, Estonian, French, German, Japanese,
+  Italian, Polish, Portuguese (Brazilian), Portuguese (Portuguese), Russian,
+  Spanish, Swedish and Turkish language support.
+- Online help system.
+- Optional anonymous message receivers.
+- Integrated support for HKP keyservers (wwwkeys.pgp.net).
+- Clears signed patch files which can still be processed by patch.
+- and many more things….
+
+## Notes
+
+- This is the modern version of GnuPG (2.2 branch). If you want the stable
+  version (2.0 branch) install [Gpg4win](https://chocolatey.org/packages/Gpg4win).
+]]></description>
+    <releaseNotes>
+* [Release Announcements](https://www.gnupg.org/index.html)
+    </releaseNotes>
+  </metadata>
+  <files>
+    <file src="tools\**" target="tools" />
```

Comment:
Add `legal` folder and move license and verifications file into it. Make sure au updater honors this.

Reply:
Well, I copied what [qbittorrent.nuspec](https://github.com/chocolatey-community/chocolatey-coreteampackages/blob/master/automatic/qbittorrent/qbittorrent.nuspec) does.